From 6df4da2a3aef71c5172b6a250c449bc267279022 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Wed, 19 Nov 2014 22:28:40 -0500
Subject: [alsa] Revert: Kconfig: rename HAS_IOPORT to HAS_IOPORT_MAP

Message-id: <1416436120-55539-1-git-send-email-jarod@redhat.com>
Patchwork-id: 100380
O-Subject: [RHEL7 PATCH] alsa: Revert: Kconfig: rename HAS_IOPORT to HAS_IOPORT_MAP
Bugzilla: 1112200
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>
RH-Acked-by: Jaroslav Kysela <jkysela@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

Bugzilla: 1112200

This reverts commit 8fbe6eb026ed37a8154dea15d55ab728a01f6f36.

This was a partial cherry-pick of an upstream change that also changed the
system-wide HAS_IOPORT to HAS_IOPORT_MAP. The problem with that is that
without that system-wide part of the change, an alsa sound driver we were
building in RHEL7.0 is now dependent on a config var that doesn't actually
exist in our tree. So lets just back this out and let lx6464es.ko build
again.

CC: Jaroslav Kysela <jkysela@redhat.com>
CC: Ivan Vecera <ivecera@redhat.com>
CC: Prarit Bhargava <prarit@redhat.com>
CC: John Linville <linville@redhat.com>
CC: Paul Bunyan <pbunyan@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/isa/Kconfig b/sound/isa/Kconfig
index 0216475..affa134 100644
--- a/sound/isa/Kconfig
+++ b/sound/isa/Kconfig
@@ -191,7 +191,7 @@ config SND_ES18XX
 
 config SND_SC6000
  tristate "Gallant SC-6000/6600/7000 and Audio Excel DSP 16"
- depends on HAS_IOPORT_MAP
+ depends on HAS_IOPORT
  select SND_WSS_LIB
  select SND_OPL3_LIB
  select SND_MPU401_UART
diff --git a/sound/pci/Kconfig b/sound/pci/Kconfig
index 69b03aa..7387eaf 100644
--- a/sound/pci/Kconfig
+++ b/sound/pci/Kconfig
@@ -685,7 +685,7 @@ config SND_LOLA
 
 config SND_LX6464ES
  tristate "Digigram LX6464ES"
- depends on HAS_IOPORT_MAP
+ depends on HAS_IOPORT
  select SND_PCM
  help
    Say Y here to include support for Digigram LX6464ES boards.
-- 
1.7.1